README.md

```diff
@@ -95,25 +95,26 @@ npm t
 ```
 
 #### End-to-end tests
-In order for tests under the make target `test-integration-coordinator` to work, you must have built and be running the coordindator locally. In a seperate shell run:
+New examples can be added in the relevant directory under `models/primary-care|secondary-care`
```

Comment:
Should this be `models/examples/primary-care|secondary-care`?

Reply:
updated